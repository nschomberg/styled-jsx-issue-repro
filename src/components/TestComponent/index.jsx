import React from "react";

import { color } from "../../constants";

const TestComponent = class extends React.Component {
  render() {
    return (
      <div>
        Test Component
        <style jsx>
          {`
            div {
              color: ${color};
            }
          `}
        </style>
      </div>
    );
  }
};

export default TestComponent;
