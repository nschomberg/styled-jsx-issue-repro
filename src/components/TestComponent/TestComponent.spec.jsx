import React from "react";
import renderer from "react-test-renderer";

import TestComponent from ".";

it("renders correctly", () => {
  const tree = renderer.create(<TestComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
