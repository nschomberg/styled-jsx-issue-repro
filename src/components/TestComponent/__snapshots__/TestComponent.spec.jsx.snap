// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="jsx-3497767100"
>
  Test Component
</div>
`;
